compiler/luci-interpreter/src/loader/KernelBuilder.test.cpp

```diff
@@ -693,6 +693,7 @@ TEST_F(KernelBuilderTest, TransposeConv)
   auto *output_shape = createInputNode();
   auto *filter = createInputNode();
   auto *input = createInputNode();
+  auto *bias = createInputNode();
```

Comment:
There should be corresponding `checkTensor` below.

Reply:
Currently, `CircleTransposeConv` doesn't have a bias term. so after update luci commented on 3, it will be updated more.

Reply:
I see. In this case this line is unnecessary.
Please add a TODO comment to not forget to update the test when luci IR has support for the bias term.

Reply:
Okay I will add it.

Reply:
Add TODO and some comments that will be comment out after luci updated.